packages/core/src/accordion/accordion-panel.element.scss

```diff
@@ -8,32 +8,52 @@
 :host {
   --border-color: #{$cds-alias-object-border-color};
   --border-width: #{$cds-alias-object-border-width-100};
-  --transition-duration: 200ms;
+  --animation-duration: #{$cds-global-animation-duration-secondary};
+  --animation-easing: #{$cds-global-animation-easing-primary};
 }
 
 .accordion-header-button {
   background: #{$cds-alias-object-opacity-0};
-  border: 0;
+  border: 0 none;
   padding: 0;
   width: 100%;
 }
 
 .accordion-content {
-  height: 0;
-  transition: transform height var(--transition-duration);
-  transition-timing-function: cubic-bezier(0, 0.99, 0, 0.99);
   overflow-y: hidden;
+  box-shadow: 0 calc(-1 * var(--border-width)) 0 0 var(--border-color);
```

Comment:
Had to use a box shadow for the content border because border was causing the animation to nudge out at the very end.